src/servers/http/v1/launcher.rs

```diff
@@ -73,7 +81,7 @@ impl Launcher {
 
         let cloned_handle = handle.clone();
 
-        tokio::task::spawn_local(async move {
+        tokio::task::spawn(async move {
             shutdown_signal.await;
             cloned_handle.shutdown();
         });
```

Comment:
Hi @da2ce7 I think one tests was failing because of the `spwan_local`but only in the runner. See https://github.com/torrust/torrust-tracker/issues/521. Maybe it does not fail after changing this.